### ISSUE TITLE
typo in doc/core/howto/endpoints.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5431]
|[<img alt="warner's avatar" src="https://avatars.githubusercontent.com/u/27146?s=50" width="50" height="50">](https://github.com/warner)| @warner reported|
|-|-|
|Trac ID|trac#5431|
|Type|defect|
|Created|2011-12-11 20:31:47Z|

The endpoints HOWTO has a small-but-confusing typo: it recommends using (a non-sensical) ```reactor.callback(30, attempt.cancel)``` instead of the real ```reactor.callLater```.

Patch included.

Attachments:

* [endpoints-doc.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c44/c443b5377b509e990dca447059509706e034cd0c/cde61c52057df9bd2449d43fb88b64742ab751a2.diff) (408 bytes) - added by warner on 2011-12-11 20:32:21Z - patch to fix the typo

<details><summary>Searchable metadata</summary>

```
trac-id__5431 5431
type__defect defect
reporter__warner warner
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1323635507000000 1323635507000000
changetime__1323636768000000 1323636768000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks, looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [33362]) Fix simple example error in the endpoints howto

Author: warner
Reviewer: exarkun
Fixes: [#5431](https://github.com/twisted/twisted/issues/5431)
